### PR TITLE
[DOR-81]: Temporarily hide records without thumbnails from search results

### PR DESCRIPTION
**Pull request by janehughes**
Ticket URL: https://national-archives.atlassian.net/browse/DOR-81

## About these changes

Hides any results which don't contain a thumbnail. This is a temporary fix for user testing, to hide parent records in our subset of record results which don't contain thumbnails. 

**Note:** this will need to be reverted after user testing (or at an agreed time) to ensure we don't unintentionally hide any records without thumbnails in future (see ticket https://national-archives.atlassian.net/browse/DOR-82)

## How to check these changes

Where possible, provide guidance to help your reviewer

## Before assigning to reviewer, please make sure you have

- [ ] Checked things thoroughly before handing over to reviewer
- [ ] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [ ] Ensured that PR includes only commits relevant to the ticket
- [ ] Waited for all CI jobs to pass before requesting a review
- [ ] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -10,3 +10,6 @@ KONG_CLIENT_VERIFY_CERTIFICATES=True
 KONG_CLIENT_TEST_MODE=False
 KONG_CLIENT_TEST_FILENAME=records.json
 PLATFORMSH_CLI_TOKEN=your-api-token-here
+API_CLIENT_NAME_PREFIX=
```
Comment (ahosgood) [inline .env.example:13]: Could we pre-populate this example with the prefix we are using?

**Comment by ahosgood** [comment]
Is this meant to be merged into `develop`?

File: .env.example (hunk 1 above)
Comment (janehughes) [reply .env.example]: Yep - probably worth doing a separate MR for that to avoid confusion. I'll put another one in for it.

**Comment by janehughes** [comment]
> Is this meant to be merged into `develop`?

Nope 🤦‍♀️  updated! thanks :)

**Review by ahosgood — APPROVED** [review]

**Pull request merged**